src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [currentMovie, setCurrentMovie] = useState<Movie | null>(null);
+  const [searchError, setSearchError] = useState(false);
+
+  const handleSearch = async () => {
+    if (title) {
+      const movieFromServer = await getMovie(title);
+
+      setSearchError(movieFromServer.Response === 'False');
+
+      setCurrentMovie(movieFromServer);
+    }
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  const handleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setSearchError(false);
+  };
+
+  const handleAddToList = () => {
+    if (currentMovie) {
+      addMovie(currentMovie);
+      setTitle('');
+      setCurrentMovie(null);
+    }
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              value={title}
+              onChange={handleInput}
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={cn('input', { 'is-danger': !title })}
```

Comment:
I think you should use `searchError` instead of `!title` because you input has red borders right from the start and after adding the movie to the list